spring-web/src/main/java/org/springframework/http/converter/ResourceHttpMessageConverter.java

```diff
@@ -147,31 +145,14 @@ protected boolean supportsRepeatableWrites(Resource resource) {
 
 
 	protected void writeContent(Resource resource, HttpOutputMessage outputMessage)
-			throws IOException, HttpMessageNotWritableException {
-
-		// We cannot use try-with-resources here for the InputStream, since we have
```

Comment:
its seems to be just a try catch around the resource operation. 

BUILD SUCCESSFUL 

<img width="1082" alt="image" src="https://github.com/user-attachments/assets/54935ce6-33d8-4763-bd1e-ed3434e2709f" />